src/app.module.ts

```diff
@@ -20,13 +20,15 @@ import { TagsModule } from './tags/tags.module';
     TypeOrmModule.forRoot({
       type: 'postgres',
       url: process.env.DATABASE_URL,
+      namingStrategy: new SnakeNamingStrategy(),
```

Comment:
`SnakeNamingStrategy`'s been used to make sure that all of the columns' names are in a snake convention because I kept facing this bug:
```js
QueryFailedError: column treeEntity.parentId does not exist
```
it didn't go even when I renamed the column name as:
```typeorm
  @JoinColumn({ name: 'parent_id' })
```
but it went away when I used this package